tests/cli-tests/__snapshots__/artifacts.test.ts.snap

```diff
@@ -84,7 +74,7 @@ export type TodoUpdateArgs = {
 }
 
 export type TodoCreateInput = {
-  readonly title?: Scalars['String'] | null
+  readonly title?: string | null
```

Comment:
This is easier on the eyes from a developer documentation point of view

Reply:
I have been annoyed by this scalar in generating types to be used with separate frontend app.